runtime/layouts/imix.ld

```diff
@@ -0,0 +1,9 @@
+/* Layout for the Hail board, used by the examples in this repository. */
```

Comment:
This comment still says Hail, presumably it should say Imix.

Reply:
Fixed